sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContentEventEmailAddresses.md

```diff
@@ -0,0 +1,65 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontenteventemailaddresses
+applicable: SharePoint Online
+title: Get-SPOContentEventEmailAddresses
+schema: 1.0.0
+author: mshabaz
+ms.author: mshabaz
+ms.reviewer:
+manager: miabhish
+---
+
+# Get-SPOContentEventEmailAddresses
+
+## SYNOPSIS
+
+Retrieves email addresses associated with a specific content event category. If no category is specified by the user, email addresses for all categories of content events will be provided.
+
+## SYNTAX
+
+```powershell
+Get-SPOContentEventEmailAddresses [-Category] <ContentEventCategory>
+```
+
+## DESCRIPTION
+
+Retrieves email addresses associated with a specific content event category. If no category is specified by the user, email addresses for all categories of content events will be provided.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Get-SPOContentEventEmailAddresses -Category Ransomware
+```
+
+In Example 1, the system will display the email addresses categorized under ransomware that the admin had previously entered.
+
+## PARAMETERS
+
+### -Category
+
+Specifies the Content event category.
```

Comment:
>Content event category [](http://example.com/codeflow?start=14&length=22)

what's the right casing? e.g., as compared to lines 18 or 28?

Reply:
same to the other files

Reply:
done